src/main.cpp

```diff
@@ -1,13 +1,36 @@
-#include "dht22.hpp"
+#include "dht_11.hpp"
+#include "dht_22.hpp"
 #include "wrap-hwlib.hpp"
+// Need because temperature contains a pure virutal function1 see:
+// https://stackoverflow.com/questions/920500/what-is-the-purpose-of-cxa-pure-virtual for more information.
+extern "C" void __cxa_pure_virtual();
+extern "C" void __cxa_pure_virtual() {
+    while (true) {
+    }
+}
 
 int main() {
```

Comment:
Please cleanup the main function.
If this is supposed to be an example, please write it down on the wiki.